### PR TITLE
WIP: upgrade to jakarta-servlet 6.0 and jetty 11

### PR DESCRIPTION
**Pull request by mmelko**
Prerequisite for this is: https://github.com/jolokia/jolokia/pull/540

Quarkus and SB needs to be updated too.

### CODE REVIEW

File: hawtio-system/pom.xml [hunk 1]
```diff
@@ -15,17 +15,23 @@
   </properties>
 
   <dependencies>
+<!--    <dependency>-->
+<!--      <groupId>jakarta.servlet</groupId>-->
+<!--      <artifactId>jakarta.servlet-api</artifactId>-->
+<!--      <version>${servlet-api-version}</version>-->
+<!--      <scope>provided</scope>-->
+<!--    </dependency>-->
```
Comment (tadayosi) [inline hawtio-system/pom.xml:23]: Can we remove this?

File: hawtio-system/pom.xml [hunk 2]
```diff
@@ -15,17 +15,23 @@
...
     <dependency>
-      <groupId>javax.servlet</groupId>
-      <artifactId>javax.servlet-api</artifactId>
-      <version>${servlet-api-version}</version>
+      <groupId>jakarta.servlet</groupId>
+      <artifactId>jakarta.servlet-api</artifactId>
+      <version>6.0.0</version>
```
Comment (tadayosi) [inline hawtio-system/pom.xml:27]: Let's keep using `${servlet-api-version}`.

File: hawtio-system/pom.xml [hunk 3]
```diff
@@ -15,17 +15,23 @@
...
       <scope>provided</scope>
     </dependency>
 
     <dependency>
       <groupId>org.jolokia</groupId>
-      <artifactId>jolokia-core</artifactId>
-      <version>${jolokia-version}</version>
+      <artifactId>jolokia-core-jakarta</artifactId>
+      <version>2.0.0-SNAPSHOT</version>
```
Comment (tadayosi) [inline hawtio-system/pom.xml:34]: Let's keep using `${jolokia-version}`.

File: hawtio-system/src/main/java/io/hawt/web/filters/FlightRecordingDownloadFacade.java [hunk 4]
```diff
@@ -180,11 +180,11 @@ public String getRequestURI() {
...
-        @SuppressWarnings("deprecation")
-        @Override
-        public String getRealPath(String path) {
-            return replaceInPath(super.getRealPath(path));
-        }
+//        @SuppressWarnings("deprecation")
+//        @Override
+//        public String getRealPath(String path) {
+//            return replaceInPath(super.getRealPath(path));
+//        }
```
Comment (tadayosi) [inline hawtio-system/src/main/java/io/hawt/web/filters/FlightRecordingDownloadFacade.java:187]: Don't comment out, just remove it.

File: hawtio-system/src/main/java/io/hawt/web/proxy/ProxyServlet.java [hunk 5]
```diff
@@ -280,8 +280,8 @@ protected void service(HttpServletRequest servletRequest, HttpServletResponse se
             // Pass the response code. This method with the "reason phrase" is deprecated, but it's the only way to pass the
             //  reason along too.
             //noinspection deprecation
-            servletResponse.setStatus(statusCode, proxyResponse.getStatusLine().getReasonPhrase());
-
+            servletResponse.setStatus(statusCode);
+           // servletResponse.set
```
Comment (tadayosi) [inline hawtio-system/src/main/java/io/hawt/web/proxy/ProxyServlet.java:284]: What's this comment?  Can we remove it?

**Review by tadayosi — COMMENTED** [review]
A few minor comments, but overall it looks good!

**Comment by mmelko** [comment]
@tadayosi thanks for the feedback!

**Review by tadayosi — APPROVED** [review]

**Pull request merged**